### PR TITLE
Add redirecto for Contracts 3.0 Crowdsale deprecation

### PR DESCRIPTION
**Pull request by nventuro**
Spawned from https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2116

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -58,3 +58,6 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 # Removed top level GSN section
 /openzeppelin/gsn/getting-started      /learn/sending-gasless-transactions 301
 /openzeppelin/gsn/getting-started.html /learn/sending-gasless-transactions 301
+
+# Redirect Crowdsale API to deprecation page
+/contracts/3.x/api/crowdsale /contracts/3.x/crowdsale 301
```
Comment (frangio) [inline _redirects]:
```suggestion
/contracts/3.x/api/crowdsale /contracts/3.x/crowdsales 301
```

Comment (nventuro) [reply _redirects]: It looks like the API is singular (https://docs.openzeppelin.com/contracts/2.x/api/crowdsale) but the guide is plural (https://docs.openzeppelin.com/contracts/2.x/crowdsales)

Comment (frangio) [reply _redirects]: Yes this is the change I suggested.

Comment (nventuro) [reply _redirects]: Ah I misread the diff lol. Thanks!

**Pull request merged**